book/views.py

```diff
@@ -1,3 +1,32 @@
-from django.shortcuts import render
+import json
+from datetime         import timedelta, date
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models          import Book
+
+
+class CommingSoonView(View):
```

Comment:
```suggestion
class CommingSoonBookView(View):
```
데이터 이름을 넣어서 좀 더 명확히 하시는 것을 추천드립니다 👍

Reply:
@soheon-lee 
수정했습니다. 감사합니다.